app/js/actions/labOrders/labConceptsAction.js

```diff
@@ -0,0 +1,33 @@
+import {
+  FETCH_LAB_CONCEPTS_SUCCESS,
+  FETCH_LAB_CONCEPTS_FAILURE,
+} from '../actionTypes';
+import axiosInstance from '../../config';
+import loading from '../loading';
+
+export const fetchLabConceptsSuccess = response => ({
+  type: FETCH_LAB_CONCEPTS_SUCCESS,
+  payload: response,
+});
+
+export const fetchLabConceptsFailure = error => ({
+  type: FETCH_LAB_CONCEPTS_FAILURE,
+  payload: error,
+});
+
+export const fetchLabConcepts = conceptUUID => (dispatch) => {
+  dispatch(loading('FETCH_LAB_CONCEPTS', true));
+  return axiosInstance.get(`/concept/${conceptUUID}`)
+    .then((response) => {
+      if (response.status !== 200) {
+        throw Error(response.statusText);
+      }
```

Comment:
why are you throwing an error here, and what would be handling the thrown error, since the reducers are not going to handle it in this case.

Reply:
Hi @Efosaok.
_`...why are you throwing an error here...`_:  It's only when the status code is `200` that I am sure that the response from the server is valid otherwise it's not. Axios does not know about this so I have to find a way around it and that is why I am throwing an error if the status code is not `200`.
_`...what would be handling the thrown error...`_: There is catch block below that does this.